README.md

```diff
@@ -16,24 +16,39 @@
 
 ### 1. Install
 
-* Linux/macOS: 
-    ```bash 
-    curl -sSL https://raw.githubusercontent.com/zeabur/cli/main/hack/install.sh | sh
-    ```
-* Windows: go to [release page](https://github.com/zeabur/cli/releases) to download the latest version.
+> Using script
 
-(TIP: you can put the binary file in the PATH environment variable to use it conveniently.)
+* Linux/macOS:
+
+```bash
+curl -sSL https://dub.sh/zeabur | sh
+```
+
+* Windows
+
+```powershell
+iwr -useb https://dub.sh/zb-win | iex
+```
+
+#### **(then restart your powershell)**
```

Comment:
The four-level directory here seems a bit unusual, it feels like it separates the context. Would bold formatting be enough?

Reply:
OK, No Problem